trusted_os/flash.go

```diff
@@ -240,18 +312,18 @@ func flashFirmware(storage Card, t FirmwareType, elf []byte, pb config.ProofBund
 		return err
 	}
 
-	log.Printf("SM flashing %s config (%d bytes) @ 0x%x", t, len(confEnc), confBlock)
+	// Log firmware bytes first before updating config so that in case of any error the unit
```

Comment:
I found this comment a bit confusing, should it be "Flash firmware bytes first..." instead of "Log firmware bytes first..."?

Reply:
d'oh, yes - thanks!